### PR TITLE
Doc. Clean documentation of ipv4 ipv6 from domains

### PR DESCRIPTION
**Pull request by den-crane**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

**Comment by robot-ch-test-poll** [comment]
This is an automated comment for commit 2923d57757cd28dde82d8edd762c1912129a68e4 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/50554/2923d57757cd28dde82d8edd762c1912129a68e4/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

**Comment by den-crane** [comment]
Wondering, how to debug `You are trying to create client-side redirections to invalid paths.`
I don't see any errors in my redirects.

**Comment by den-crane** [comment]
It seems it fails because of clickHouse-doc repo.

**Comment by rschu1ze** [comment]
@yakov-olkhovskiy

File: docs/en/sql-reference/data-types/ipv6.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-slug: /en/sql-reference/data-types/domains/ipv6
+slug: /en/sql-reference/data-types/ipv6
 sidebar_position: 60
 sidebar_label: IPv6
 ---
 
 ## IPv6
 
-`IPv6` is a domain based on `FixedString(16)` type and serves as a typed replacement for storing IPv6 values. It provides compact storage with the human-friendly input-output format and column type information on inspection.
+IPv6 addresses. Stored in 16 bytes as UInt128.
```
Comment (yakov-olkhovskiy) [inline docs/en/sql-reference/data-types/ipv6.md:9]:
```suggestion
IPv6 addresses. Stored in 16 bytes as UInt128 big-endian.
```

**Comment by rfraposa** [comment]
FYI - here is the error:

```
These paths are redirected to but do not exist:
- /en/sql-reference/data-types/domains/ipv4
- /en/sql-reference/data-types/domains/ipv6
```

And here are the relevant redirects in [ClickHouse/clickhouse-docs/docusaurus.config.js](https://github.com/ClickHouse/clickhouse-docs/blob/main/docusaurus.config.js):

```
					{
						from: '/en/data_types/domains/ipv4',
						to: '/en/sql-reference/data-types/domains/ipv4',
					},
					{
						from: '/en/sql_reference/data_types/domains/ipv4',
						to: '/en/sql-reference/data-types/domains/ipv4',
					},
					{
						from: '/en/data_types/domains/ipv6',
						to: '/en/sql-reference/data-types/domains/ipv6',
					},
					{
						from: '/en/sql_reference/data_types/domains/ipv6',
						to: '/en/sql-reference/data-types/domains/ipv6',
					},
```

**Comment by rfraposa** [comment]
I understand what this PR is doing, but it's not "fixing" anything that is broken - all the links work and are based on what appears to be historical URLs.

@den-crane - did you need these changes for some reason?

**Comment by den-crane** [comment]
@rfraposa there are datatypes domains in Clickhouse. For example before 23.1 IPv6 was such datatype - it was a domain based on FixedString. It behaved as FixedString, it had traits of FixedString. It was placed in the special path in the documentation `/data-types/domains` to emphasize its nature. After 23.1 IPv6 became native type, it's not domained datatype anymore.

**Comment by rfraposa** [comment]
Got it - good to know. Thanks!

I will try to update docusaurus.config.js so that your PR will build successfully.

**Comment by DanRoscigno** [comment]
I will pick this one up in the morning

**Pull request merged**